src/main/deltachat.js

```diff
@@ -117,19 +117,21 @@ class DeltaChatController extends events.EventEmitter {
       log.warning(warning)
     })
 
-    dc.on('DC_EVENT_ERROR', (error) => {
-      this.emit('DC_EVENT_ERROR', error)
+    const onError = error => {
+      this.emit('error', error)
       log.error(error)
+    }
+
+    dc.on('DC_EVENT_ERROR', (error) => {
+      onError(error)
     })
 
     dc.on('DC_EVENT_ERROR_NETWORK', (first, error) => {
-      this.emit('DC_EVENT_ERROR_NETWORK', first, error)
-      log.error(error)
+      onError(error)
     })
 
     dc.on('DC_EVENT_ERROR_SELF_NOT_IN_GROUP', (error) => {
-      this.emit('DC_EVENT_SELF_NOT_IN_GROUP', error)
-      log.error(error)
+      onError(error)
```

Comment:
Time to refactor this now since we had three events doing the same thing. We emit `'error'` from the delta chat controller, and the ui should present all this to the user regardless of what type of error it was.